service/controller/resource/monitoring/prometheus/resource.go

```diff
@@ -320,20 +296,6 @@ func toPrometheus(ctx context.Context, v interface{}, config Config) (metav1.Obj
 			key.EtcdSecret(config.Installation, cluster),
 		}
 
-		prometheus.Spec.RuleSelector = &metav1.LabelSelector{
-			MatchExpressions: []metav1.LabelSelectorRequirement{
-				{
-					Key:      key.ClusterTypeKey,
-					Operator: metav1.LabelSelectorOpNotIn,
-					Values:   []string{"workload_cluster"},
-				},
-				{
-					Key:      key.TeamLabel,
```

Comment:
Do we want to execute customer rules ?

Reply:
Mimir will take care of rule evaluation

Reply:
We have to wait for mimir before merging this PR, right ?

Reply:
Oh yes yes yes

Reply:
So should you set this PR in draft for the moment ?

Reply:
I cannot

Reply:
Well I do not see the button